arches/app/views/auth.py

```diff
@@ -183,24 +186,25 @@ def post(self, request):
                 return redirect(confirmation_link)
 
             admin_email = settings.ADMINS[0][1] if settings.ADMINS else ""
-            email_context = {
-                "button_text": _("Signup for Arches"),
-                "link": confirmation_link,
-                "greeting": _(
-                    "Thanks for your interest in Arches. Click on link below \
+            email_context = return_message_context(
+                 _(
```

Comment:
ah, this style of interpolation only works with f-strings, and f-strings don't work with i18n. Currently these strings are not being interpolated at all.

```
(Pdb) email_context
{'greeting': 'Thanks for your interest in {settings.APP_NAME}. Click on link below
...}
```
To end up with correct interpolation, can these strings be altered to [user the format() method?](https://docs.python.org/3/library/stdtypes.html#str.format)

This should apply to all interpolated strings throughout this PR